modules/ossm-threescale-manifests.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/threescale-adapter.adoc
+
+[id="ossm-threescale-manifests_{context}"]
+= Generating manifests from a deployed adapter
+To generate manifests from a deployed adapter, run the following command replacing the following arguments with the 3scale Admin Portal URL, Service ID, and access token:
+
+----
+$ oc exec -n istio-system $(oc get po -n istio-system -o jsonpath='{.items[?(@.metadata.labels.app=="3scale-istio-adapter")].metadata.name}') \
+-it -- ./3scale-config-gen \
+--url="https://replace-me.3scale.net:443" --service="example-service-id" --token="access-token"
+----
+
+This will produce sample output to the terminal and a unique identifier (UID) that must be used for multiple service use cases. Edit these samples if required and create the objects using the `oc create` command.
+
+https://docs.openshift.com/container-platform/3.11/servicemesh-install/servicemesh-install.html#routing-service-traffic-through-the-adapter[Update the workload] with the required annotations:
```

Comment:
Is this still the best link? Can you provide the rest of the information in the assembly?

Reply:
Fixed the link.